translate/storage/placeables/__init__.py

```diff
@@ -45,14 +45,14 @@
 Please refer to the XLIFF specification to get a better understanding.
 """
 
-import base
-import interfaces
-import general
-import xliff
-from base import *
-from base import __all__ as all_your_base
-from strelem import StringElem
-from parse import parse
+from . import base
+from . import interfaces
+from . import general
+from . import xliff
```

Comment:
Would be really necessary to import these modules to export them again in this module `__all__`? I ask because you will be accessing them using `translate.storage.placeables.base` anywhere.